src/core/jsonschema/official_resolver.in.cc

```diff
@@ -54,8 +60,10 @@ auto sourcemeta::core::schema_official_resolver(std::string_view identifier)
 
     // Just for compatibility given that this is such a common issue
   } else if (identifier == "https://json-schema.org/draft/2020-12/schema#") {
-    return sourcemeta::core::parse_json(
+    auto schema = sourcemeta::core::parse_json(
         R"EOF(@METASCHEMA_JSONSCHEMA_2020_12@)EOF");
+    schema.at("$id").into(sourcemeta::core::JSON{identifier});
```

Comment:
We should never mutate the official meta-schemas, as they are supposed to be immutable. What was the intention?